docroot/themes/custom/mass_theme/templates/includes/molecules-header-search.html.twig

```diff
@@ -8,21 +8,24 @@
 {% block content %}
 
 <section class="ma__header-search">
-  <form action="https://search.mass.gov/" class="ma__form js-header-search-form">
-    <div role="search" class="ma__suggestions-container js-suggestions-container ma__search-banner__form {{ searchFormId }}">
-      <div class="visually-hidden js-suggestions-help" role="status"  aria-live="assertive"></div>
+  <form role="search" action="https://search.mass.gov/" class="ma__form js-header-search-form">
+    <div class="ma__suggestions-container js-suggestions-container ma__search-banner__form {{ searchFormId }}">
+      <div id="autosuggest-info" class="ma__visually-hidden" aria-hidden="true">Suggestions are presented as an open option list only when they are available.</div>
+      <div class="ma__visually-hidden js-suggestions-help" role="status"  aria-live="assertive"></div>
       <label for="header-search" class="ma__header-search__label">Search terms</label>
       <input
-          id="header-search"
-          name="q"
-          class="ma__header-search__input"
-          autocomplete="off"
-          aria-autocomplete="both"
-          role="combobox"
-          aria-own="suggestions-list"
-          placeholder="Search Mass.gov"
-          data-suggest=""
-          type="text" />
+        id="header-search"
+        name="q"
+        class="ma__header-search__input"
+        type="search"
+        inputmode="search"
+        autocomplete="list"
+        role="combobox"
+        aria-expanded="false"
+        aria-controls="suggestions-list"
+        placeholder="Search Mass.gov"
+        data-suggest=""
+        aria-describedby="autosuggest-info" />
       <div class="ma__suggestions" data-suggestions=""></div>
```

Comment:
```suggestion
      <div class="ma__suggestions" data-suggestions=""></div>
```

curious if the `data-suggestions` attribute is being used in anyway. I don't see the value change when there's value in the listbox

Reply:
removing this seems to affect the autocomplete functionality. tabling this for now